gradle-scalatest.plugin

```diff
@@ -0,0 +1,21 @@
+buildscript {
+	repositories {
+		maven {
+			url "https://plugins.gradle.org/m2/"
+		}
+	}
+	dependencies {
+		classpath "com.github.maiflai:gradle-scalatest:0.8"
+	}
+}
+
+if (!project.plugins.findPlugin(com.github.maiflai.ScalaTestPlugin)) {
```

Comment:
Not a technical downside as far as I'm aware, but at least as far as Gradle 2.x is concerned it's definitely not a conventional means of applying plugins which has the potential to confuse those less than familiar with Gradle